docs/source/sci_api_tutorial/assign_mutant.rst

```diff
@@ -0,0 +1,193 @@
+==============================================
+Assign Mutant
+==============================================
+
+Briefs
+==============================================
+The target for this tutorial to mutate the target protein at defined residue(s). The main function provided are ``assign_mutant()`` and ``mutate_stru()``, in which ``assign_mutant()`` assigns mutants targeted in the study. Decode the user assigned ``{pattern}`` based on the ``{stru}`` and get a list of mutants defined by a list of mutation objects each.
+
+Input/Output
+==============================================
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/mutation_assign_mutant_io.svg
```

Comment:
the output should be mutants as List[List[Mutatioon]]

Reply:
I like how you use different color to highlight the 2 components of the module. Could it be a better idea to have the API that is not discussed in the page as gray? (because user tend to forgot the color mapping as it is across 2 pages.)

Reply:
Also the output of the mutate_stru should be `mutant_structure` (it is just a name but will make it more clear)
You can reference QZ's branch to see how to make this diagram more clear to users.